plugins/databricks/personal_access_token.go

```diff
@@ -0,0 +1,118 @@
+package databricks
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func PersonalAccessToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:    credname.PersonalAccessToken,
+		DocsURL: sdk.URL("https://docs.databricks.com/dev-tools/auth.html"),
+		// The management URL is spesific to each deployment of Databricks, but this
+		// link describes how to manage tokens provided you are logged in.
+		ManagementURL: sdk.URL("https://docs.databricks.com/dev-tools/auth.html"),
```

Comment:
ReadMe is in a simular situation where the management URL are specific/different for each user. We discussed how to handle this situation for the `ManagementURL` in this thread: https://github.com/1Password/shell-plugins/pull/106#discussion_r1049357649. I propose we do the same here:
- Don't set the optional `ManagementURL` field, the product will fall back to using the DocsURL with slightly different wording.
- Possible future extensions might support wildcards for the ManagementURL so it can support ReadMe, Databricks and others that have dynamic management URLs.

To help us improve & smoothen the contribution process, @bsamseth was it clear for you the ManagementURL field is optional?

Reply:
Got it, removed in 9d5f612b6127a9f1664ba054b86f1947d4fa917d. 

It was not immediately clear that it was optional from the template it self. However, a quick goto-definition on the field shows me a clear comment, so that's equally on me. It didn't occur to me to check though, so perhaps the template wording should/could be changed to make this more clear.